<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomSettingsDialog</class>
 <widget class="QDialog" name="CustomSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agent Custom Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbDisablePanel">
        <property name="text">
         <string>Disable Panel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDisableHUD">
        <property name="text">
         <string>Disable HUD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDisableAntiAliasHUD">
        <property name="text">
         <string>Disable Anti-Alias HUD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEnableClouds">
        <property name="text">
         <string>Enable Clouds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEnableClouds3D">
        <property name="text">
         <string>Enable Clouds 3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEnableFullscreen">
        <property name="text">
         <string>Enable Fullscreen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEnableTerraSync">
        <property name="text">
         <string>Enable Terra Sync</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEnableRealWeatherFetch">
        <property name="text">
         <string>Enable Real Weather Fetch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Field of View:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leFOV"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>View Offset:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leViewOffset">
        <property name="toolTip">
         <string>Specify the default forward view direction in degrees. Increments of 50-60 degrees are suggested.</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Additional Args:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lwAdditionalArgs"/>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbAddCustomArg">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbRemoveCustomArg">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
